emails/models.py

```diff
@@ -36,6 +36,17 @@ class CannotMakeAddressException(Exception):
     pass
 
 
+'''
+class ReplyAddress(models.Model):
+    address = models.CharField(
+        max_length=64, default=address_default, unique=True
+    )
+    sender = models.CharField(
```

Comment:
note: this will capture the sender's `Reply-To` so maybe this column should be `sender_reply_to` ?